src/utils/url/drello-api.ts

```diff
@@ -0,0 +1,20 @@
+import { serverEnv } from "src/constants/system";
+import { currentEnv } from "src/utils/server-env";
+
+export const domain = () => {
+  switch (currentEnv()) {
+    case serverEnv.development:
```

Comment:
Will you return separate URLs for development and production later depending on the backend API's URL?